test/test-markdownlint.sh

```diff
@@ -16,7 +16,9 @@ if [ -z "$MDL" ]; then
 	fail_test "The 'mdl' utility can't be found."
 fi
 
-STYLE=$($mktemp)
+# create a tmp file in current directory e.g. "./tmp.rMKAQ63Cm6"
+# so that mdl command can locate the file (must contain a '/' or end in '.rb')
+STYLE=$($mktemp -p .)
```

Comment:
Why is this change needed?

Reply:
After updating mdl to 0.13.0 locally, the test passes as well in my dev env. This change is not necessary and has been removed from the patch.